src/js/components/Anchor/doc.js

```diff
@@ -103,4 +103,6 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
+  ...themeDocUtils.focusStyle,
+  ...themeDocUtils.normalizeColor('The text color used inside the Anchor.'),
```

Comment:
`normalizeColor` is used for the color of the icon within Anchor. https://github.com/grommet/grommet/blob/726f4f4abdb8050f1de70a9b5c69793aadb628a0/src/js/components/Anchor/Anchor.js#L45